debian/patches/dsa/check_running_kernel_zstd_ubuntu_fix

```diff
@@ -0,0 +1,23 @@
+Index: pkg-nagios-plugins-contrib/dsa/checks/dsa-check-running-kernel
+===================================================================
+--- dsa-check-running-kernel.dist	2022-03-12 21:36:28.000000000 +0100
++++ dsa-check-running-kernel	2022-03-12 21:36:32.000000000 +0100
+@@ -162,6 +162,8 @@
+ 	cat_vmlinux "$image" "\x02\x21\x4c\x18"  "lz4 -dc"  0
+ 	# lzo compressed image
+ 	cat_vmlinux "$image" "\x89\x4c\x5a\x4f\x00\x0d\x0a\x1a"  "lzop -dc"  0
++	# zstd compressed image
++	cat_vmlinux "$image" "\x28\xb5"              "zstd -d" 0
```

Comment:
### Introduction
For us, it was sufficient to scan for a two-byte header here in order to satisfy our main aim, to make the program work on machines with ZSTD-compressed Linux kernels.

### Walkthrough
```shell
grep -abo $'\x28\xb5' /boot/vmlinuz-5.13.19-5-pve | head -n1
17100:(�
```

```shell
dd if=/boot/vmlinuz-5.13.19-2-pve bs=17100 skip=1 | zstd -d | strings | grep "Linux version"
Linux version 5.13.19-2-pve (build@proxmox) (gcc (Debian 10.2.1-6) 10.2.1 20210110, GNU ld (GNU Binutils for Debian) 2.35.2) #1 SMP PVE 5.13.19-4 (Mon, 29 Nov 2021 12:10:09 +0100) ()
zstd: /*stdin*\: unsupported format
```

### Evaluation
However, you can see this emits a warning like `zstd: /*stdin*\: unsupported format`. Maybe this can be improved by adjusting the `header` pattern further?

Currently, this does not do any harm to the functionality of `check_running_kernel`.

Reply:
I've extended the scan to a four-byte header `\x28\xb5\x2f\xfd`, so I am resolving this.